modular_skyrat/modules/customization/modules/client/augment/organs.dm

```diff
@@ -79,7 +79,7 @@
 	slot = AUGMENT_SLOT_TONGUE
 
 /datum/augment_item/organ/tongue/normal
-	name = "Organic tongue"
+	name = "Human tongue"
```

Comment:
I don't see this as necessarily being needed, considering how many species use the normal human tongue by default.

Organic tongue feel like it'd fit just fine.

Reply:
Fixed by commit 09f34d23f945d2f7094bcdc55df4f98df81a2670